clang-tools-extra/docs/clang-tidy/checks/misc/use-internal-linkage.rst

```diff
@@ -25,3 +25,16 @@ Example:
   }
   // already declared as extern
   extern int v2;
+
+Options
+-------
+
+.. option:: FixMode
+
+  Selects the fix mode when fixing automatically.
```

Comment:
Not sure if it is better: `Selects what kind of a fix the check should provide`?